src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1815,137 +2357,160 @@ void func_80A153FC(EnGo* this, PlayState* play) {
     }
 }
 
-s32* func_80A15684(EnGo* this, PlayState* play) {
-    static s32 D_80A16704[] = {
-        D_80A16100,
-        D_80A16164,
+/**
+ * Return the message script appropriate for the actor
+ *
+ * @returns Message Script corresponding to the actor
+ */
+s32* EnGo_GetMessageScript(EnGo* this, PlayState* play) {
+    static s32 sMsgScriptGraveBros[] = {
+        sMsgScriptGoronGravemaker,
+        sMsgScriptGoronBrother,
     };
 
-    if (this->unk_3EC != 0) {
-        return D_80A1640C;
-    }
-
-    if (ENGO_GET_F(&this->actor) == ENGO_F_1) {
-        switch (ENGO_GET_70(&this->actor) % 6) {
-            case ENGO_70_0:
-                return D_80A163BC;
-            case ENGO_70_1:
-                return D_80A163CC;
-            case ENGO_70_2:
-                return D_80A163DC;
-            case ENGO_70_3:
-                return D_80A163EC;
-            case ENGO_70_4:
-                return D_80A163FC;
-            case ENGO_70_5:
-                return D_80A163FC;
+    if (this->sleepState != ENGO_AWAKE) {
+        return sMsgScriptGoronSleeping; // Lullaby Hint
+    }
+
+    if (ENGO_GET_TYPE(&this->actor) == ENGO_STRETCHER) {
+        switch (ENGO_GET_SUBTYPE(&this->actor) % 6) {
+            case ENGO_STRETCHER_A:
+                return sMsgScriptGoronStretcherA;
+            case ENGO_STRETCHER_B: // Unused!
+                return sMsgScriptGoronStretcherB;
+            case ENGO_STRETCHER_C:
+                return sMsgScriptGoronStretcherC;
+            case ENGO_STRETCHER_D:
+                return sMsgScriptGoronStretcherD;
+            case ENGO_STRETCHER_HAMSTRINGSIT:
+                return sMsgScriptGoronStretcherHamstring;
+            case ENGO_STRETCHER_HAMSTRINGSTAND:
+                return sMsgScriptGoronStretcherHamstring;
         }
```

Comment:
Same comment as before regarding blank lines, and providing it matches, we try to make sure every switch has a default.

Reply:
Unable to get a matching compile when using a default case.